prismic-model/json/series.json

```diff
@@ -1,10 +1,9 @@
 {
   "Article series": {
-    "title": {
-      "type": "StructuredText",
+    "name": {
```

Comment:
When do we use 'name' or 'title'?

Reply:
We always use title.
Except in the past.